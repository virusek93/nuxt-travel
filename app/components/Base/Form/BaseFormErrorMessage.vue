<script lang="ts" setup></script>

<template>
  <span class="text-sm text-red-500"><slot /></span>
</template>
