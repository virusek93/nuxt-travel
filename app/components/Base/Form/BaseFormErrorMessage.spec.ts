import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import BaseFormErrorMessage from '~/components/Base/Form/BaseFormErrorMessage.vue'

const slots = {
  default: 'Test slot',
}

describe('BaseFormErrorMessage', () => {
  it('Should render and match the snapshot', () => {
    const wrapper = mount(BaseFormErrorMessage, { slots })
    expect(wrapper.element).toMatchSnapshot()
  })
  it('Should render slots data', () => {
    const wrapper = mount(BaseFormErrorMessage, { slots })
    expect(wrapper.text()).toBe(slots.default)
  })
})
