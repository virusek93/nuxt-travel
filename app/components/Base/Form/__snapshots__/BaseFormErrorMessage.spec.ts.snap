// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BaseFormErrorMessage > Should render and match the snapshot 1`] = `
<span
  class="text-sm text-red-500"
>
  
  Test slot
  
</span>
`;
